apis.md.go

```diff
@@ -44,6 +44,19 @@ func (c *WorkwxApp) execUserList(req reqUserList) (respUserList, error) {
 	return resp, nil
 }
 
+// execUseridByMobile 手机号获取 userid
+func (c *WorkwxApp) execUseridByMobile(req reqUseridByMobile) (respUseridByMobile, error) {
```

Comment:
This function should be auto-generated, instead of manually coded like this. See `docs/apis.md` and `dummy_for_generate.go` for details.

Reply:
fixed.